src/asynch/ws.rs

```diff
@@ -36,6 +36,7 @@ mod error {
         PostcardError(postcard::Error),
         #[cfg(feature = "prost")]
         ProstError(ProstError),
+        OversizedFrame(usize, usize),
```

Comment:
How about switching this to `OversizedFrame(usize)` where size is the delta? You anyways use only the delta. Or if you want to keep both sizes (and having the error enum getting so big) use the structural syntax `OversizedFrame { .. }`

Reply:
I think just having the delta makes perfect sense because the user of the lib will have defined the N constant themselves, and if they want a more verbose error message they can use delta to create it. 

Updated.